arraysCycles/primeNumber.js

```diff
@@ -0,0 +1,11 @@
+// 9. Реализовать цикл, который выводит в консоль простые числа
+
+let number = 15;
+
```

Comment:
Not fully solved. If we will start from `i = -3` this loop prints negative numbers, 0 and 1, but should not do it.